<launch>
    <!-- setting up ROS server parameters -->
    <arg name="robotAsistance" default= "1.0"/>
    <arg name="pathWait" default= "60"/>
    <param name="robotAsistanceParam" value="$(arg robotAsistance)" />
    <param name="pathWaitParam" value="$(arg pathWait)" />

    <!-- starting ROS package nodes -->
    <node pkg="assigment_rt_3" type="master_Robot.py" name="master_Robotxxx" output="screen"/>
    <node pkg="assigment_rt_3" type="UI_Robot.py" name="UI_Robot" output="screen"/>
    <!-- <remap from="cmd_vel" to="remmaped_cmd_vel"/> -->
    <!-- <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" launch-prefix="konsole -e" output="screen"/> -->
    <include file="$(find assigment_rt_3)/launch/launchTeleop.launch">
    </include>
</launch>